tests/test_crash_reports.py

```diff
@@ -47,20 +44,20 @@ def test_that_top_crasher_filter_all_return_results(self, mozwebqa):
         product = csp.header.current_product
         cstc = csp.header.select_report('Top Crashers')
         if cstc.results_found:
-            Assert.equal(product, cstc.page_heading_product)
+            assert product == cstc.page_heading_product
 
         cstc.click_filter_by('All')
-        Assert.greater(cstc.results_count, 0)
+        assert cstc.results_count > 0
```

Comment:
This is fine, we're only defining an order when doing equality comparisons. This is the most readable way to do this assert.